src/Bindings/LuaState.cpp

```diff
@@ -845,6 +843,21 @@ bool cLuaState::GetStackValue(int a_StackPos, float & a_ReturnedVal)
 
 
 
+bool cLuaState::GetStackValue(int a_StackPos, int & a_Value)
+{
+	double tmp;
+	if (GetStackValue(a_StackPos, tmp))
+	{
+		a_Value = FloorC(tmp);
+		return true;
+	}
+	return false;
+}
```

Comment:
This is not needed at all, there's already a templated function handling `int`s:
https://github.com/cuberite/cuberite/blob/master/src/Bindings/LuaState.h#L268-L287